proof/access-control/Syscall_AC.thy

```diff
@@ -397,20 +400,13 @@ lemma handle_reply_respects:
                    dest: invs_cur)
   done
 
-lemma ethread_set_time_slice_pas_refined[wp]:
-  "ethread_set (tcb_time_slice_update f) thread \<lbrace>pas_refined aag\<rbrace>"
-  apply (simp add: ethread_set_def set_eobject_def | wp)+
-  apply (clarsimp simp: pas_refined_def tcb_domain_map_wellformed_aux_def)
-  apply (erule_tac x="(a, b)" in ballE)
-   apply force
-  apply (erule notE)
-  apply (erule domains_of_state_aux.cases, simp add: get_etcb_def split: if_split_asm)
-   apply (force intro: domtcbs)+
-  done
+
 
 lemma thread_set_time_slice_pas_refined[wp]:
-  "thread_set_time_slice tptr time \<lbrace>pas_refined aag\<rbrace>"
-  by (simp add: thread_set_time_slice_def | wp)+
+  "\<lbrace>pas_refined aag and pspace_aligned and valid_vspace_objs and valid_arch_state\<rbrace>
```

Comment:
This precondition (and those of several similar lemmas) could be weakened using the changes from https://github.com/seL4/l4v/pull/837/commits/357ccc987dee33b7b9780145e17f3205d462ceda. I'll do so once that commit is either in master or is able to be cherry-picked into this branch.

Reply:
does this also apply to `lemma pas_refined`?